### PR TITLE
gives cmo a tablet and a pair of medical sunglasses

### PR DESCRIPTION
**Pull request by boodaliboo**
bounty: https://forums.yogstation.net/threads/5-add-tablet-and-sunglasses-to-cmo-loadout-roundstart-ss13-server.22354/
:cl:  
rscadd: gave cmo sunglasses and a tablet
/:cl:

### CODE REVIEW

**Comment by Tipy1802** [comment]
I feel like the sunglasses should have the chemistry hud not the health one since the CMO already gets an implant with the health hud and a cmo actually needs to see the contents of a beaker

**Comment by boodaliboo** [comment]
not sure what the error is

**Comment by Jarod1200** [comment]
> I feel like the sunglasses should have the chemistry hud not the health one since the CMO already gets an implant with the health hud and a cmo actually needs to see the contents of a beaker

Chem vision sunglasses are called beer goggles. You can easily craft a pair with science goggles, sunglasses, cables, screwdriver, and wirecutters

**Comment by Takahiru** [comment]
Damn too fast bood a free five bucks

File: code/modules/jobs/job_types/chief_medical_officer.dm [hunk 1]
```diff
@@ -47,7 +47,7 @@
 	suit = /obj/item/clothing/suit/toggle/labcoat/cmo
 	l_hand = /obj/item/storage/firstaid/regular
 	suit_store = /obj/item/flashlight/pen
-	backpack_contents = list(/obj/item/melee/classic_baton/telescopic=1) //yogs - removes med budget
+	backpack_contents = list(/obj/item/melee/classic_baton/telescopic=1, /obj/item/clothing/glasses/hud/health/sunglasses=1, /obj/item/modular_computer/tablet=1) //yogs - removes med budget
```
Comment (SomeguyManperson) [inline code/modules/jobs/job_types/chief_medical_officer.dm]: use glasses variable for the glasses so they start equipped
```suggestion
	glasses = /obj/item/clothing/glasses/hud/health/sunglasses
	backpack_contents = list(/obj/item/melee/classic_baton/telescopic=1, /obj/item/modular_computer/tablet=1) //yogs - removes med budget
```

**Comment by Firewolf34** [comment]
Can you ensure the tablet starts with the "Lifeline" app and whatever associated necessary hardware

Pretty important for CMO job. Good PR.

**Review by Amelia0010 — APPROVED** [review]

**Pull request merged**